fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroupMgr.java

```diff
@@ -114,6 +114,32 @@ public List<TPipelineResourceGroup> getResourceGroup(String groupName) throws Us
         return resourceGroups;
     }
 
+    public QueryQueue getResourceGroupQueryQueue(String groupName) throws UserException {
+        readLock();
+        try {
+            ResourceGroup resourceGroup = nameToResourceGroup.get(groupName);
+            if (resourceGroup == null) {
+                throw new UserException("Resource group " + groupName + " does not exist");
+            }
+            return resourceGroup.getQueryQueue();
+        } finally {
+            readUnlock();
+        }
+    }
+
+    // called after group loaded from image
+    // currently mainly used to init Resource Group's query queue
+    public void initResourceGroup() {
```

Comment:
It might be better to call this function in gsonPostProcess()

Reply:
Good idea.